keps/prod-readiness/sig-auth/4601.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 4601
 alpha:
   approver: "@wojtek-t"
+beta:
```

Comment:
@liggitt Sorry - didn't get on time for reviewing it before it merged :)

I just have a couple nits:
1) The upgrade/downgrade section is not filled in. I know it's somewhat obvious in this case, but it would benefit adding a sentence there

2) Feature enablement/disablement tests
 Can you please link them?

3) upgrade->downgrade->upgrade tests
 Can you please be slightly more specific on the test performed? (e.g. cluster setup, what cel expression, how exactly it was tested (what requests were send to validate it) etc.?